src/pages/project.tsx

```diff
@@ -1,6 +1,7 @@
+import React from "react";
 import { Project } from "../Components/index";
 
-const project = () => {
+const project: React.FC = () => {
   // {/* /* visible은 모달이 보이는 지 안 보이는 지, state는 모달 상태
```

Comment:
need comment at this?

Reply:
I got it.
However, it doesn't matter if delete it now?

Is there any other reason I don't know?